cli/tests/integration/init_tests.rs

```diff
@@ -128,3 +128,41 @@ fn init_subcommand_with_quiet_arg() {
   assert_contains!(output.stdout(), "1 passed");
   output.skip_output_check();
 }
+
+#[test]
+fn init_subcommand_with_existing_file() {
+  let context = TestContextBuilder::new().use_temp_cwd().build();
+  let cwd = context.temp_dir().path();
+
+  let testdata_init_dir = context.testdata_path().join("init");
+  let testdata_init_main_ts = testdata_init_dir.join("main.ts");
+  let cwd_main_ts = cwd.join("main.ts");
+  testdata_init_main_ts.copy(&cwd_main_ts);
+
+  let output = context.new_command().args("init").split_output().run();
+
+  output.assert_exit_code(0);
+
+  let stderr = output.stderr();
+  assert_contains!(stderr, "Skipped creating main.ts as already exists");
+  assert_contains!(stderr, "Project initialized");
+  assert!(!stderr.contains("cd"));
+  assert_contains!(stderr, "deno run main.ts");
+  assert_contains!(stderr, "deno task dev");
+  assert_contains!(stderr, "deno test");
+
+  assert!(cwd.join("deno.json").exists());
+
+  let output = context
+    .new_command()
+    .env("NO_COLOR", "1")
+    .args("run main.ts")
+    .split_output()
+    .run();
+
+  output.assert_exit_code(0);
+  assert_eq!(
+    output.stdout().as_bytes(),
+    b"Log from main.ts that already exists\n"
+  );
```

Comment:
There is an assertion helper to do this. I think it's `output.assert_contains("Log from main.ts that already exists")`

Reply:
This line was copied from the other tests in this file, but I can switch this test to using that. 👍🏻

Reply:
`output.assert_contains` didn't appear to exist, however `output.assert_matches_text` did. I've switched to using it for that assertion in 62946ff22.

Reply:
Should I rewrite the other `assert_contains!` in the tests to use `assert_matches_text`? I think it requires an exact match though.

Reply:
I pushed 2c6e0a6 which changes to use it. The test is clearer. I can revert that one commit though if preferred no.